assets/wix/files.wxs

```diff
@@ -1218,6 +1215,12 @@
         <Component Id="cmp7C72179331DD48BCB12BF205F34B76AD">
           <File Id="filF65735DC9B8B4957B27F5B33D2A322F4" KeyPath="yes" Source="$(var.ProductSourcePath)\ref\System.Runtime.InteropServices.JavaScript.dll" />
         </Component>
+        <Component Id="cmpBB2921B987554B68B26862747C3BD398">
```

Comment:
We should refresh this file from the publish pipeline

Reply:
I agree. What do I do with this file, then?  The CI build keeps saying it doesn't match, and I can't get it right locally.

Reply:
I'll fix this file for you in next commit